strutil/quantity/quantity.go

```diff
@@ -200,3 +202,256 @@ func FormatDuration(dt float64) string {
 
 	return FormatAmount(uint64(dt), 4) + years
 }
+
+// Duration is represented in nano seconds.
+type Duration uint64
```

Comment:
Changed to time.Duration.

Unfortunately the go library does not provide days and years, so I still define those in terms of time.Duration. The purpose of the patch is to replace the existing formatting function, so I still want to provide days and years as before, as this may be useful in other case (such as age or snap etc...). The next step will be to move the other use cases over to use this generic renderer.